packages/services/src/contents/index.ts

```diff
@@ -118,7 +118,7 @@ export namespace Contents {
   /**
    * A contents file format.
    */
-  export type FileFormat = 'json' | 'text' | 'base64';
+  export type FileFormat = 'json' | 'text' | 'base64' | 'null';
```

Comment:
Can we use an actual null, rather than the string null? An actual null in js signifies the "lack of a value".

Reply:
absolutely

Reply:
done